server/service/src/sync/test/test_data/master_list_line.rs

```diff
@@ -11,7 +11,7 @@ const MASTER_LIST_LINE_1: (&str, &str) = (
     "item_ID": "item_a",
     "imprest_quan": 0,
     "order_number": 1,
-    "price": 3.14
+    "price": 3.15
```

Comment:
My clippy was angry red... 👀

Reply:
because of .14 ? What kind of clippy setting is this ?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/approx_constant

Reply:
Ohh lol, it thinks we are meant to be doing Pi ?

Reply:
Yup 🤣